source/src/molecular-unfolding/cdk/stack-main.ts

```diff
@@ -76,6 +76,14 @@ export class MainStack extends SolutionStack {
       constraintDescription: 'Any printable ASCII character ranging from the space character (\u0020) through the end of the ASCII character range',
     });
 
+    const quickSightRoleArn = new CfnParameter(this, 'QuickSightRoleArn', {
+      type: 'String',
+      description: 'QuickSight IAM Role Arn, find it from https://us-east-1.quicksight.aws.amazon.com/sn/admin#aws',
+      allowedPattern: 'arn:aws:iam::[0-9]{12}:role/[^/]*',
```

Comment:
pls check the doc to validate the role name, such as length and characters

Reply:
add Constraints